Software/app/src/main/java/org/foi/air/smartcharger/validations/RegistrationValidations.kt

```diff
@@ -0,0 +1,31 @@
+package org.foi.air.smartcharger.validations
+
+class RegistrationValidations {
+
+    private val emailPattern = Regex("^[^@\\s]+@[^@\\s]+\\.[^@\\s]+$", RegexOption.IGNORE_CASE)
+
+    fun firstNameValidation(firstName: String): Boolean{
+        if(firstName.isNotEmpty()) return true
```

Comment:
You can just write 'return firstName.isNotEmpty()', which would result in cleaner code that works exactly the same way. Same goes for all other functions in this class.

Reply:
Thank you for the insight. I rewrote the functions soo that they look cleaner.